setup.py

```diff
@@ -31,4 +31,5 @@
         "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
```

Comment:
Can you update this list? = start from 3.8 I guess

Reply:
Sure.